cargo-near/src/abi.rs

```diff
@@ -53,15 +55,34 @@ pub(crate) fn write_to_file(
     if !generate_docs {
         strip_docs(&mut contract_abi);
     }
-    let near_abi_json = serde_json::to_string(&contract_abi)?;
-    let out_path_abi = crate_metadata
-        .target_directory
-        .join(crate_metadata.root_package.name.replace('-', "_") + "_abi.json");
-    fs::write(&out_path_abi, near_abi_json)?;
+    let near_abi_serialized = match format {
+        AbiFormat::Json => serde_json::to_vec_pretty(&contract_abi)?,
+        AbiFormat::JsonMin => serde_json::to_vec(&contract_abi)?,
+    };
+    let near_abi_compressed = match compression {
+        AbiCompression::NoOp => near_abi_serialized,
+        AbiCompression::Zstd => zstd::encode_all(near_abi_serialized.as_slice(), 0)?,
```

Comment:
Nope, thanks for pointing it out. I have tried out max level (22) compression and it shaves FT ABI size from 914 bytes down to 826 bytes. zstd documentation says that >=20 levels require more memory, but I wouldn't expect this to matter in our case, so we should be safe to use it. Just for a reference point, with level 15 FT ABI size is 830 bytes. So there is a minuscule difference, but it might of course differ on a case-to-case basis.